modules/end-user-guide/pages/devfile-introduction.adoc

```diff
@@ -0,0 +1,25 @@
+[id="introduction-to-devfile_{context}"]
+= Introduction to devfile in {prod}
+
+link:https://devfile.io/[Devfiles] are `yaml` text files used for development environment customization. Use them to configure a devfile to suit your specific needs and share the customized devfile across multiple workspaces to ensure identical user experience and build, run, and deploy behaviours across your team.
+
+.{prod}-specific devfile features
+
+.Devfile and Universal Developer Image
+
+A devfile isn't required to start a workspace. If you don't include a devfile in your project repository, {prod} automatically loads a default devfile with a Universal Developer Image (UDI).
```

Comment:
Avoid contractions and make it more user focused. What do you think of these rewrites? 

`You do not need a devfile to start a workspace. If you do not include a devfile in your project repository, {prod} automatically loads a default devfile with a Universal Developer Image (UDI).`